mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java

```diff
@@ -157,6 +158,11 @@ public void setCredentials(Map<String, UsernamePasswordCredentials> credentials)
      * Absolute (from domain name) URL path where the gml service can be called
      */
     public static final String GML_URL = DOC_URL + "gml/";
+
+    /**
+     * Absolute (from domain name) URL path where the fe service can be called
+     */
+    public static final String FE_URL = DOC_URL + "fe/";
```

Comment:
Should the code https://github.com/jdenisgiguere/georchestra/blob/FilterBuilder_save_load/mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java#L127-L166 be moved before the first public method declaration?